interfaces/builtin/ofono.go

```diff
@@ -288,6 +288,15 @@ ATTRS{idVendor}=="1c9e", ATTRS{idProduct}=="9605", ENV{ID_USB_INTERFACE_NUM}=="0
 LABEL="ofono_speedup_end"
 `
 
+const ofonoPermanentSlotUDevTag = `
+KERNEL=="tty[A-RT-Z]*[0-9]|cdc-wdm[0-9]*", TAG+="###SLOT_SECURITY_TAGS###"
+KERNEL=="tun",         TAG+="###SLOT_SECURITY_TAGS###"
+KERNEL=="tun[0-9]*",   TAG+="###SLOT_SECURITY_TAGS###"
+KERNEL=="dsp",         TAG+="###SLOT_SECURITY_TAGS###"
+KERNEL=="ttyLTM[0-9]", TAG+="###SLOT_SECURITY_TAGS###"
+KERNEL=="ttyUSB[0-9]", TAG+="###SLOT_SECURITY_TAGS###"
```

Comment:
If modem is always a symlink, then that's fine (we could remove the apparmor rule then too since apparmor resolves symlinks).